krabs/krabs/schema_locator.hpp

```diff
@@ -94,16 +95,11 @@ namespace krabs {
 
     /**
      * <summary>
-     * Fetches and caches schemas from TDH. Also has a singleton
-     * instance. It is preferable to inject the locator, but the
-     * current krabs implementation isn't built in a way that
-     * allows the schema_locator to be injected in all of the
-     * places where Schemas or Parsers are constructed.
+     * Fetches and caches schemas from TDH.
```

Comment:
```suggestion
     * Fetches and caches schemas from TDH.
     * NOTE: this cache also reduces the number of managed to native transitions
     * when krabs is compiled into a managed assembly.
```

Reply:
This important comment was lost in as part of the review:
 https://github.com/microsoft/krabsetw/pull/96/files/6ae1e9234571f370df03b8d8dbffa62419a21cc0#r371471476